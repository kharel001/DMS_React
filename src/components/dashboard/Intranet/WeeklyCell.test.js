import React from 'react';
import WeeklyCell from './WeeklyCell';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const dummy_data={
    data:{
        id:1,
        projectName:"project1",
        sales:400,
        income:100000,
         growth:2
    }
}
const setup = (props = {}) => {
    return shallow( < WeeklyCell {...props}  />
    )
}


it("WeeklyCell",()=>{
    const wrapper= setup(dummy_data);
    expect(wrapper).toMatchSnapshot();
})