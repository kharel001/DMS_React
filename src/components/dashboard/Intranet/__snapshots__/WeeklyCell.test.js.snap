// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeeklyCell 1`] = `ShallowWrapper {}`;
